examples/manifests/sagemaker-notebook-modules.yaml

```diff
@@ -0,0 +1,36 @@
+
+name: notebook
+path: modules/sagemaker/sagemaker-notebook
+parameters:
+  - name: notebook_name
+    value: dummy
+  - name: instance_type
+    value: ml.t2.xlarge
+  - name: direct_internet_access
+    value: Enabled
+  - name: root_access
+    value: Disabled
+  - name: volume_size_in_gb
+    value: 20
+  - name: imds_version
+    value: 2
+  - name: subnet_ids
+    valueFrom:
+      moduleMetadata:
+        group: networking
+        name: networking
+        key: PrivateSubnetIds
+  - name: vpc_id
+    valueFrom:
+      moduleMetadata:
+        group: networking
+        name: networking
+        key: VpcId
+  - name: kms_key_arn
+    value: arn:aws:kms:us-east-1:xxxxxx:key/xxxxx-xxxx-xxxx-xxxxx
+  - name: code_repository
+    value: https://git-codecommit.us-east-1.amazonaws.com/v1/repos/xxxxxxxxx
```

Comment:
For the purpose of being able to use this for a one-click deployment, can we remove all `xxx` values please? It's fine if it creates it's own KMS key / doesn't have a repo association.